code/exceptions/RealMeException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class RealMeException extends Exception
+{
+    const INVALID_IDENTITY_VALUE = 0;
```

Comment:
Using sublcasses is more expressive and clearer what's going on IMO... I'm not sure there's any documented standard on exception classes vs codes but it's pretty normal in popular libraries for there to be different exception classes - I've not really ever seen this approach